lms/djangoapps/courseware/tests/__init__.py

```diff
@@ -106,7 +106,7 @@ def setup_course(self):
         # username = robot{0}, password = 'test'
         self.users = [
             UserFactory.create()
-            for i in range(self.USER_COUNT)
+            for dummy0 in range(self.USER_COUNT)
```

Comment:
Pylint supports this syntax as a valid name for unused variables.

Using `dummyN` makes it clearer that it is a throwaway value:
- `_` shadows `gettext`
- `__` closely resembles the above

Often times, I'll prepend a `_` to a signal an usused parameter, but
`_i` is too short and a bit more cryptic and unclear.